datahub-web-react/src/app/preview/DefaultPreviewCard.tsx

```diff
@@ -260,9 +260,13 @@ export default function DefaultPreviewCard({
     };
 
     const shouldShowRightColumn = (topUsers && topUsers.length > 0) || (owners && owners.length > 0);
+    const uniqueOwnerUrns = new Set();
+    const uniqueOwners = owners?.filter(
+        (owner) => !uniqueOwnerUrns.has(owner.owner.urn) && uniqueOwnerUrns.add(owner.owner.urn),
+    );
```

Comment:
Good use of a Set 👍 

Optional: Maybe a `useUniqueOwners(owners)` custom hook? Lots of logic already in this file.

Reply:
hm why hook as opposed to a utility function? since it's not using any hooks itself - I could just do `getUniqueOwners` in a utility file